### PR TITLE
Added PHP×BRU (Brussels, Belgium)

### PR DESCRIPTION
**Pull request by voidgraphics**
Been wanting to get some meetups started in Belgium. This looks like the perfect opportunity!

I am admittedly new at this, so I'm looking forward to the getting started resources you guys are planning, as well as tips from more experienced organizers.

The phpxbru.com should be proxied to your IP already, I just finished setting things up in Cloudflare.

Let me know if I missed anything!

✌️

### CODE REVIEW

File: groups.json [hunk 1]
```diff
@@ -11,5 +11,13 @@
...
-	}
+	},
+    "phpxbru.com": {
+        "external": false,
+        "name": "PHP×BRU",
+        "region": "Brussels, Belgium",
+        "description": "PHP & Laravel meetups for belgian developers",
+        "timezone": "Europe/Brussels",
+        "bsky_url": "https://bsky.app/profile/phpxbru.com"
+    },
```
Comment (princejohnsantillan) [inline groups.json:22]: I don't think trailing commas are allowed in json.

**Comment by inxilpro** [comment]
I know there are at least 2 if not 3 meetups in Brussels—but I'm not sure if any of them have had meetings since the pandemic. Have you already looked around? You can use the resources on https://phpx.world/organizers to make sure there aren't other groups before you start a new one…

**Comment by voidgraphics** [comment]
Ah I missed that page about the resources to check, my bad!

Looks like there's a "Laravel Brussels" that's been inactive since 2019, and a "BrusselsPHP" which has a lot of followers on meetup but looks like there hasn't been an event since June 2023. Could still be active and interested though.

I'll get in contact with both to see if they want to collaborate on this!

**Comment by inxilpro** [comment]
Awesome. Just move this out of draft when you're ready to move forward!

**Pull request merged**